# Request
int32 starting_num
---
# Result
bool is_finished
---
# Feedback
int32 current_num